### PR TITLE
✅ Add unit tests for turtle and svg + dxf + gcode saver

### PR DESCRIPTION
**Pull request by coliss86**

### CODE REVIEW

**Comment by coliss86** [comment]
Unit test are KO is github action, maybe a `\n` problem...

**Comment by coliss86** [comment]
ok fixed now

File: src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCodeTest.java [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
 public class LoadGCodeTest {
 
-    @BeforeEach
-    public void beforeEach() {
+    @BeforeAll
+    public static void beforeEach() {
```
Comment (i-make-robots) [inline src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCodeTest.java:21]: beforeAll() ?

File: src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadScratch3Test.java [hunk 2]
```diff
@@ -17,8 +17,8 @@
 
 public class LoadScratch3Test {
 
-    @BeforeEach
-    public void beforeEach() {
+    @BeforeAll
+    public static void beforeEach() {
```
Comment (i-make-robots) [inline src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadScratch3Test.java:21]: here, too.

File: src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCodeTest.java (hunk 1 above)
Comment (coliss86) [reply src/test/java/com/marginallyclever/makelangelo/makeArt/io/vector/LoadGCodeTest.java]: yes correct !

**Comment by coliss86** [comment]
need to update after 73c7abe67bcb1620337c35633fc135a8327be950

**Pull request merged**